local_pathfinding/ompl_path.py

```diff
@@ -156,9 +200,8 @@ def _init_simple_setup(self) -> og.SimpleSetup:
         simple_setup.setOptimizationObjective(objective)
 
         # set the planner of the simple setup object
-        # TODO: implement and add planner here
-        planner = og.RRTstar(space_information)
-        simple_setup.setPlanner(planner)
+        # TODO: implement and add planner
+        simple_setup.setPlanner(self.choose_planner(self.planner, space_information))
```

Comment:
Keep the same structure as before (2 separate lines, it's clearer), just remove the todo